src/lib/database/index.js

```diff
@@ -81,24 +80,14 @@ class Database {
 
   async refreshSchema () {
     const getSchemaSql = `
-      SELECT name, sql
-      FROM sqlite_master
-      WHERE type='table' AND name NOT LIKE 'sqlite_%';
+    WITH columns as (SELECT a.tbl_name, json_group_array(json_object('name', b.name,'type', b.type)) as column_json 
```

Comment:
It's expected that when a type is not defined it should say `N/A`.
It can be achived with IIF like this:
`json_object('name', b.name,'type', IIF(b.type = '', 'N/A', b.type)`

Reply:
Glad this approach works; added conditional.